content/news/20.4networking-rework.md

```diff
@@ -0,0 +1,231 @@
+---
+title: "The Networking Refactor"
+date: 2023-12-31T21:00:00+00:00
+publishDate: 2023-12-31T21:00:00+00:00
+categories:
+- News
+author: oriononline
+summary: |
+    This post gives an overview of the changes made to the networking system of NeoForge 20.4.
+description: |
+    This post gives an overview of the changes made to the networking system of NeoForge 20.4.
+---
+
+# Introduction
+## Reworked networking
+Welcome to the reworked networking pull request for NeoForged.
```

Comment:
"welcome to the pull request"? This is a blog post. Also, the project is NeoForge, 
NeoForged is the team.